### PR TITLE
Make logger trim to terminal size by default

### PR DESCRIPTION
**Pull request by cmaujean**
Option --no-trim to get "full" output

Signed-off-by: Christopher Maujean <christopher@maujean.org>

### CODE REVIEW

**Comment by erikh** [comment]
Can one of the admins verify this patch?

**Comment by erikh** [comment]
Ok to test

File: main.go [hunk 1]
```diff
@@ -97,25 +101,24 @@ func main() {
...
-		log := logger.New("main")
-
 		if len(args) < 1 {
 			cli.ShowAppHelp(ctx)
 			log.Error("Please provide a filename to process!")
 			os.Exit(1)
 		}
 
-		log = logger.New(args[0])
+		log = logger.New(args[0], notrim)
```
Comment (erikh) [inline main.go]: this can be pruned

File: multi/multi_test.go [hunk 2]
```diff
@@ -122,7 +122,7 @@ func mkBuilders(plans map[int]string) []*builder.Builder {
 	builders := []*builder.Builder{}
 
 	for i := range plans {
-		l := logger.New("")
+		l := logger.New("", true)
```
Comment (erikh) [inline multi/multi_test.go]: why is this true here and false in all the other testing spots?

Comment (cmaujean) [inline multi/multi_test.go]: To see if you were paying attention... ;) I'll fix it.

**Comment by cmaujean** [comment]
Docs added.

**Comment by erikh** [comment]
Needs a rebase

**Comment by cmaujean** [comment]
closed by #198

**Pull request closed without merging**